xml/System.Numerics/INumberBase`1.xml

```diff
@@ -852,7 +852,7 @@ This function treats both positive and negative zero as zero and so will return
         <param name="y">The value to compare with <paramref name="x" />.</param>
         <summary>Compares two values to compute which has the greater magnitude.</summary>
         <returns>
-          <paramref name="x" /> if it is greater than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
+          <paramref name="x" /> if it has the greater magnitude than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
```

Comment:
Should this be `a` instead of `the`?
```suggestion
          <paramref name="x" /> if it has a greater magnitude than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
```

Reply:
I believe `a` is correct here and `the` is correct in the linked example. I'm not an English major and couldn't tell you "why" though